x11-misc/xplanet/xplanet-1.3.1-r1.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit flag-o-matic
+
+DESCRIPTION="Render images of the earth into the X root window"
+HOMEPAGE="http://xplanet.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ppc ~ppc64 ~sparc ~x86"
+IUSE="gif jpeg png tiff truetype X"
+
+RDEPEND="
+	gif? ( media-libs/giflib:= )
+	jpeg? ( virtual/jpeg:* )
+	png? (
+		media-libs/libpng:0
+		media-libs/netpbm
+		)
+	tiff? ( media-libs/tiff:0 )
+	truetype? (
+		media-libs/freetype:2
+		x11-libs/pango
+		)
+	X? (
+		x11-libs/libX11
+		x11-libs/libXScrnSaver
+		x11-libs/libXext
+		x11-libs/libXt
+		)
+"
+DEPEND="
+	${RDEPEND}
+	truetype? ( virtual/pkgconfig )
+	X? (
+		x11-proto/scrnsaverproto
+		x11-proto/xproto
+		)
+"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-giflib.patch
+	"${FILESDIR}"/${P}-remove-null-comparison.patch
+)
+
+src_configure() {
+	# econf says 'checking pnm.h presence... no'
+	use png && append-cppflags -I/usr/include/netpbm
+
+	local myconf
+
+	use X \
+		&& myconf+=" --with-x --with-xscreensaver" \
```

Comment:
could you make `myconf` a bash array and use that instead? arrays are less retarded than whitespace separated strings